packages/runner-ct/src/app/RunnerCt.tsx

```diff
@@ -218,6 +230,21 @@ const App: React.FC<AppProps> = observer(
         <KeyboardHelper />
       )
 
+    const MainAreaComponent: React.FC | typeof SplitPane = props.state.spec
```

Comment:
This fixes the second bug described in the PR description. We should not render the Reporter <SplitPane> at all if there is no spec selected. It makes resizing buggy.